test/functional/wallet_descriptor.py

```diff
@@ -60,43 +60,43 @@ def run_test(self):
         addr = self.nodes[0].getnewaddress("", "legacy")
         addr_info = self.nodes[0].getaddressinfo(addr)
         assert addr_info['desc'].startswith('pkh(')
-        assert_equal(addr_info['hdkeypath'], 'm/44\'/1\'/0\'/0/0')
+        assert_equal(addr_info['hdkeypath'], 'm/44h/1h/0h/0/0')
```

Comment:
This is an API break? (Albeit a small one)

Reply:
I suppose it would be better if legacy wallet RPC calls don't change behavior. I'll look into this (and the other comments above).

Reply:
The bigger break was actually in the wallet dump format. I've added handling to keep `hdkeypath` the same for legacy wallets.